File: sast-js/case/completeness/single_app_tracing/expression/type_cast/type_cast_003_T.js
// evaluation information start
// real case = false
// evaluation item = 完整度->单应用跟踪完整度->表达式->类型转换
// scene introduction = 隐式类型转换
// level = 2
// bind_url = completeness/single_app_tracing/expression/type_cast/type_cast_003_T
// evaluation information end
const { execSync } = require('child_process');


function type_cast_003_T(__taint_src) {
  let result = __taint_src == 5;
  __taint_sink(result);
}

function __taint_sink(o) {
  execSync(String(o));
}

const taint_src = 5;

type_cast_003_T(taint_src);

File: sast-go/cases/completeness/single_app_tracing/expression/type_cast/type_cast_003_T/type_cast_003_T.go

// evaluation information start
// real case = true
// evaluation item = 完整度->单应用跟踪完整度->表达式->类型转换
// scene introduction = 类型断言
// level = 2
// bind_url = completeness/single_app_tracing/expression/type_cast/type_cast_003_T/type_cast_003_T
// evaluation information end

package type_cast_003_T

func type_cast_003_T(__taint_src interface{}) {
	result, ok := __taint_src.(string)
	_ = ok
	__taint_sink(result)
}

func __taint_sink(o interface{}) {
}
